style.css

```diff
@@ -0,0 +1,88 @@
+.sort {
+    width: 8px;
+}
+
+.heart {
+    width: 25px;
+}
+
+.user {
+    width: 22px;
+    opacity: 0.7;
+}
+
+/* .heart:hover {
+    height: 30px;
+    width: 25px;
+    background-color: #bbb;
+    border-radius: 50%;
+    display: inline-block;
+  } */
+
+nav{
```

Comment:
@Afrocentriccynic The contents of the `nav` don't seem to sit/align properly  inside it when I add a border to the nav.
Also there is a lot more padding on the site nav compared to yours.

<img width="1440" alt="Screenshot 2020-05-03 at 8 21 06 AM" src="https://user-images.githubusercontent.com/16891274/80908425-a0d06e80-8d17-11ea-944c-cf76b09649e3.png">
<img width="1439" alt="Screenshot 2020-05-03 at 8 24 21 AM" src="https://user-images.githubusercontent.com/16891274/80908429-a3cb5f00-8d17-11ea-87e0-b3c180c9aef6.png">